protos/merlin/transformer/spec/table.proto

```diff
@@ -75,3 +78,16 @@ enum JoinMethod {
   CROSS = 5;
   CONCAT = 6;
 }
+
+message ScaleColumn {
+  string column = 1;
+  oneof scalerConfig {
```

Comment:
Similar to `encoderType` on Encoder message, I think ScaleColumn also needs `scalerType`